swirl/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1 @@
-<manifest
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:versionCode="1"
-    package="com.mattprecious.swirl"
-    >
-  <uses-sdk android:minSdkVersion="15"/>
```

Comment:
Oops. Build file says 14. Bump it to 15?

Reply:
bumped to 15